cmake/external.cmake

```diff
@@ -150,6 +150,21 @@ function(link_libopenh264 target_name)
     endif()
 endfunction()
 
+# libSRTP
+function(link_libsrtp target_name)
+    if (TG_OWT_PACKAGED_BUILD)
+        find_package(PkgConfig REQUIRED)
+        pkg_check_modules(SRTP libsrtp2)
+        if (SRTP_FOUND)
+            target_include_directories(libsrtp INTERFACE ${SRTP_INCLUDE_DIRS})
+            target_link_libraries(libsrtp INTERFACE ${SRTP_LINK_LIBRARIES})
```

Comment:
these two lines are still wrong, they should link to the specified target and include dirs must be marked as SYSTEM

Reply:
I.e.
```suggestion
            target_include_directories(${target_name} SYSTEM PRIVATE ${SRTP_INCLUDE_DIRS})
            target_link_libraries(${target_name} PRIVATE ${SRTP_LINK_LIBRARIES})
```